src/placement/jump_map.c

```diff
@@ -870,7 +870,7 @@ jump_map_obj_place(struct pl_map *map, struct daos_obj_md *md,
 	/* Allocate space to hold the layout */
 	rc = pl_obj_layout_alloc(jmop.jmop_grp_size, jmop.jmop_grp_nr,
 				 &layout);
-	if (rc != 0) {
+	if (rc) {
```

Comment:
minor, I prefer the != 0 though the effect is the same.  It's not a boolean variable